onfhir-path/src/main/scala/io/onfhir/path/FhirPathTerminologyServiceFunctions.scala

```diff
@@ -215,8 +320,44 @@ class FhirPathTerminologyServiceFunctions(context:FhirPathEnvironment) extends A
-    insertText = "trms:lookupDisplay(<source_code>, <codeSystemUrl>, <targetColumn>)",detail = "trms", label = "trms:lookupDisplay", kind = "Function", returnType = Seq("string"), inputType = Seq())
+  @FhirPathFunction(
+    documentation = FhirPathFunctionDocumentation(
+      detail = "Return the display string in preferred language for the given code. If code or system cannot be found, or displayLanguage does not match with available ones, return empty. If code, system, or displayLanguage does not evaluate to a single string, throws an error.",
+      usageWarnings = Some(Seq("A code system with the specified URL **must** be available in the terminology service, and the mapping job must be configured to use this terminology service to use this function.")),
+      parameters = Some(Seq(
+        FhirPathFunctionParameter(
+          name = "source_code",
+          detail = "Source code to find its display value.",
+          examples = None
+        ),
+        FhirPathFunctionParameter(
+          name = "codeSystemUrl",
+          detail = "Unique code system URL from the terminology service to be used to perform the lookup operation.",
+          examples = None
+        ),
+        FhirPathFunctionParameter(
+          name = "targetColumn",
+          detail = "FHIRPath string literal providing the name of the interested column from the code system. Mostly language name.",
+          examples = Some(Seq("en", "de"))
+        )
+      )),
+      returnValue = FhirPathFunctionReturn(
+        detail = None,
+        examples = Seq(
+          "\"Probe von der Haut\""
+        )
+      ),
+      examples = Seq(
+        "trms:lookupDisplay('C12', 'http://hus.fi/CodeSystem/labResults', 'tr')"
+      )
+    ),
+    insertText = "trms:lookupDisplay(<source_code>, <codeSystemUrl>, <targetColumn>)",
+    detail = "trms",
+    label = "trms:lookupDisplay",
+    kind = "Function",
+    returnType = Seq("string"),
```

Comment:
I think we can now use FHIR_DATA_TYPES, right ? If so, please make sure that we use it every return and input type.